store/migrations/1547744983_organizations.up.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+
+CREATE TABLE organization (
+	id uuid NOT NULL PRIMARY KEY,
+	provider text NOT NULL,
```

Comment:
Given that we're [querying by  `provider` and `internal_id`](https://github.com/src-d/lookout/pull/483/files#diff-94c7f4f10a53dfdc07f94f92324cde86R263) we could add an index on both columns in order to speed up the queries.

Reply:
Ok, @smacker's issue addresses both my concerns.

Reply:
Answering both conversations here, since they are related.

You are right, we should enforce the uniqueness it at the DB level, and add indexes. I simply left the automatically generated migration file, but now that you mention it I'm not sure if we should add the missing details here in this PR, or in #316 with the other missing changes.

I will take a look and if it's straightforward (I guess it should) I'll push it here.